Woof/Woof/View/SitterFlow/ScheduleView/CreateNewWalkingView.swift

```diff
@@ -0,0 +1,87 @@
+import SwiftUI
+
+/// A view providing tools for selecting date, time, and recurrence options for
+/// creating walkings in a sitter's schedule.
+struct CreateNewWalkingView: View {
+    var body: some View {
+        VStack(spacing: 30) {
+            VStack {
+                Text(createWalkingTitle)
+                    .font(.headline)
+            }
+
+            HStack {
+                Text(startDateAndTimeForWalkingLabel)
+
+                Spacer()
+
+                DatePicker("",
+                           selection: $viewModel.startTime,
+                           in: Date()..., displayedComponents: [
+                               .date,
+                               .hourAndMinute,
+                           ])
+                           .labelsHidden()
+            }
+
+            HStack {
+                Text(durationLabel)
+
+                Spacer()
+
+                Stepper(value: $viewModel.durationInMinutes, in: 30...120, step: 30) {
+                    Text("\(viewModel.durationInMinutes) min")
```

Comment:
```suggestion
                 Text(viewModel.walkingLength)
```

WDYT about extracting text formatting into a view model.
```swift
var walkingLength: String {
        "\(durationInMinutes) min"
    }
```